variables.tf

```diff
@@ -0,0 +1,161 @@
+variable "ocm_environment" {
+  type    = string
+  default = "production"
+}
+
+# Cluster
+
+variable "openshift_version" {
+  type    = string
+  default = "4.13.13"
+  validation {
+    condition     = can(regex("^[0-9]*[0-9]+.[0-9]*[0-9]+.[0-9]*[0-9]+$", var.openshift_version))
+    error_message = "openshift_version must be with structure <major>.<minor>.<patch> (for example 4.13.6)."
+  }
+}
+
+variable "cluster_name" {
+  type = string
+}
+
+variable "private" {
+  description = "Create a private cluster"
+  type        = bool
+  default     = false
+}
+
+variable "machine_pools" {
```

Comment:
I think better to have this as list and not map
Is it possible to have more specified object instead of "any"?
The same comment for IDP